### PR TITLE
Update leadership.html

### PR DESCRIPTION
**Pull request by Shivansh2407**
## Description
Update leadership page #8216

### CODE REVIEW

**Comment by ejregithub** [comment]
@Errolyn - would you mind reviewing this PR, please?

**Comment by Errolyn** [comment]
Sure! @ejregithub

I'll take a look in a few minutes :)

File: bedrock/mozorg/templates/mozorg/about/leadership.html [hunk 1]
```diff
@@ -121,7 +121,7 @@ <h3 class="group-title" id="mgmt-corp"><a href="{{ url('foundation.moco') }}">{{
...
...list of “100 Most Influential People” under “Scientists and Thinkers”. Bloomberg listed her as one of the “25 Most Influential People on the Web”. She was honored as winner of the Anita Borg Institute’s 2009 Women of Vision Award and received the Aenne Burda Award for Creative Leadership in 2010. In 2012, Mitchell was inducted into the founding group of the <a rel="external" href="http://www.internethalloffame.org/inductees/mitchell-baker">Internet Society’s Hall of Fame</a>. In 2018 she received the Webby Lifetime Achievement Award. She also received an honorary doctorate from the Universite Catholique de Louvain in Belgium. She has written for Scientific American, the Atlantic, Quartz, CNN.com and Les Annales Des Mines and appeared on NBC’s “Meet the Press”, BBC’s “HardTalk” and "CEO Secrets", CNN’s “Global Office”, NBC’s “Weekend Today”, CNBC's "Squawk Box", Cheddar TV, France 24, the UK’s Channel 4 and NPR’s “Morning Edition". She has spoken at high-level events like Tech for Good, VivaTech, Wired NextFest, World Economic Forum, Mobile World Congress, Web Summit, Collision, AI@Oxford, Bloomberg's "Sooner Than You Think", SXSW Edu, EL PAÍS Retina, RSA and Singularity University.
```
Comment (pdehaan) [inline bedrock/mozorg/templates/mozorg/about/leadership.html]: FWIW, I think `Bloomberg's` (last sentence) is the only instance of a non-smart _single_ quote in the paragraph.

And, `"CEO Secrets"`, `"Squawk Box"`, `“Morning Edition"` (closing quote; not introduced in this PR though), and `"Sooner Than You Think"` use non-smart _double_ quotes.

Comment (Errolyn) [reply bedrock/mozorg/templates/mozorg/about/leadership.html]: @Shivansh2407 if you feel comfortable with making changes to the quotes go ahead, if not I will move it into it's own issue to be handled separately.

**Comment by Errolyn** [comment]
I'm going to go ahead and merge this.

**Pull request merged**